Marlin/src/module/stepper.cpp

```diff
@@ -2143,20 +2143,14 @@ void Stepper::init() {
     E_AXIS_INIT(5);
   #endif
 
+  set_directions();
+
   // Init Stepper ISR to 122 Hz for quick starting
   HAL_timer_start(STEP_TIMER_NUM, 122);
 
   ENABLE_STEPPER_DRIVER_INTERRUPT();
 
   sei();
-
-  // Init direction bits for first moves
-  last_direction_bits = 0
-    | (INVERT_X_DIR ? _BV(X_AXIS) : 0)
-    | (INVERT_Y_DIR ? _BV(Y_AXIS) : 0)
-    | (INVERT_Z_DIR ? _BV(Z_AXIS) : 0);
```

Comment:
This change was made with a lot of fanfare. Apparently the first direction change can fail if these aren'e set according to their inverting.

Reply:
Definitely not an intended change.  I'll check for other unintended changes and back them all out later today.